lib/webtask.js

```diff
@@ -221,6 +221,10 @@ Webtask.prototype.createCronJob = function (options, cb) {
  * @returns {Promise} A Promise that will be fulfilled with the result of inspecting the token.
  */
 Webtask.prototype.inspect = function (options, cb) {
+    if (this.sandbox.securityVersion === 'v2') {
```

Comment:
What about the discussion we were having here? https://github.com/auth0/sandboxjs/pull/29/files#r162486030

Reply:
Updated to do option 5.